sycl/test/esimd/simd.cpp

```diff
@@ -73,6 +73,16 @@ bool test_simd_bin_ops() __attribute__((sycl_device)) {
   return v0[0] == 1;
 }
 
+bool test_simd_unary_ops() __attribute__((sycl_device)) {
```

Comment:
These tests are compile-only. Please create executable test as well in llvm-test-suite. Otherwise looks good to me.